pyproject.toml

```diff
@@ -8,12 +8,24 @@ version = "5.0.0"
 authors = [{name = "Sam Schott", email = "sam.schott@outlook.com"}]
 license = {text = "MIT"}
 description = "Python library for cross-platform desktop notifications"
-keywords = ["desktop-notifier"]
+keywords = [
+    "D-Bus",
+    "Desktop Bus",
+    "desktop-notifier",
+    "notifications",
+    "Notification Center",
+    "push notifications",
+    "Toast notifications",
+]
 classifiers = [
     "Development Status :: 2 - Pre-Alpha",
```

Comment:
@samschott you might want to change this as `Pre-Alpha` could be pretty off putting to people searching around for libraries on pypi

Reply:
Good point!

Reply:
Feel free to replace it with `Development Status :: 5 - Production/Stable` in this PR.